include/dlpack/dlpack.h

```diff
@@ -6,6 +6,9 @@
 #ifndef DLPACK_DLPACK_H_
 #define DLPACK_DLPACK_H_
 
+/**
+ * \brief Compatibility with C++
```

Comment:
This wasn't documented before which led to a warning when building documentation with Doxygen.